sandy/njoy.py

```diff
@@ -951,12 +1004,17 @@ def process(
         kwargs["temp"] = temperatures[0]
         text += _groupr_input(-e, -p, -v, **kwargs)
         o = 32
-        text += _moder_input(-v, o)
+        if errorr is False:
+            text += _moder_input(-v, o)
         outputs[f"tape{o}"] = join(
             wdir,
             f"{outprefix}{tag}.gendf",
             )
     if errorr:
+        if groupr:
+            kwargs["ngout"] = -v
+            p = 0
+            outputs = {}
```

Comment:
Why are you resetting the outputs?

Reply:
The outputs must be  `errorr` text files, no `gendf` text files. If the output of `get_errorr` is a also a `gendf` file we are going againts the docstring and the division of NJOY outputs into three methods